pkg/ruler/api_test.go

```diff
@@ -38,6 +38,7 @@ func TestRuler_PrometheusRules(t *testing.T) {
 		configuredRules rulespb.RuleGroupList
 		limits          RulesLimits
 		expectedRules   []*RuleGroup
+		queryParams     string
```

Comment:
Could you please add a test for the case where `type` is neither `alert` nor `record`?

Reply:
Should be added now